http/http.go

```diff
@@ -0,0 +1,39 @@
+package http
+
+import (
+	"bufio"
+	"bytes"
+	"errors"
+	"net/http"
+	"unsafe"
+
+	"github.com/machinefi/w3bstream-wasm-golang-sdk/common"
+)
+
+func Do(req *http.Request) (*http.Response, error) {
+	req.URL.Host = "w3bstream.com"
```

Comment:
how is it used in host side?

Reply:
go http package need the Host for routing.